QuickDeployment/QuickDeploymentInternetSetup.md

```diff
@@ -0,0 +1,222 @@
+
+### Scenario 2: Purchased/Acquired Certificates from CA
+
+If you have purchased or acquired a certificate from an external Certificate Authority (CA; e.g. LetsEncrypt), this process is similar to the Domain scenario, with some slight adjustments:
+
+1. Ensure that a DNS record exists, resolving the desired FQDN from your purchased/acquired SSL certificate to its external IP address.
+1. Import your certificate into the `Local Computer\Personal\Certificates` and `Local Computer\Trusted People\Certificates` stores on the QDE server. Open the "Certificates - Local Computer" MMC snap-in by pressing the Windows key, and when the Start menu pops up, type certificates. You should now see an option under the "Settings" section that says "Manage computer certificates". Alternatively, open the Run dialog (Windows key + R) and type `certlm.msc` and click "OK". Double-check to ensure that the SSL certificate you have purchased is placed in the correct stores, otherwise this process will not succeed.
+1. Under the "Personal" store, you should see a server certificate matching the FQDN of the certificate your QDE server. Double-click on the certificate to open it, and under the details tab, copy out the `Thumbprint` value of the certificate.
+1. Run the `New-SslCertificates.ps1` script and pass the thumbprint you copied above using the `-Thumbprint` parameter. Please run the below command from a PowerShell Administrator window:
+
+    ```powershell
+    Set-ExecutionPolicy Bypass -Scope Process -Force; . C:\choco-setup\files\New-SslCertificates.ps1 -Thumbprint '<YOUR_CERT_THUMBPRINT_HERE>'
+    ```
+
+1. **[OPTIONAL]** Assuming you've acquired your SSL certificate from an already-trusted Certificate Authority, your endpoints should trust this certificate natively. If you run into any issues with this, you may need to ensure that the QDE certificate is copied to the `Local Computer\Trusted People\Certificates` store on those endpoints as well.
+
+### Scenario 3: Self-Signed SSL Certificates
+
+If you are planning to use self-signed SSL certificates to secure your communication with Nexus and CCM on QDE, then you will be required to run the `New-SslCertificates.ps1` script using the command above _at least once_. As you will be exposing these two services via the Internet, you must make sure your certificates are unique from the default ones provided to every customer on the QDE image.
+
+1. Ensure you have a valid DNS record for the `CHOCOSERVER` hostname of the QDE VM, that resolves to the external IP of QDE. If not, create a host record for the hostname `CHOCOSERVER` in your Windows HOSTS file on your endpoints. In recent versions of Windows, this is located at: `“C:\Windows\System32\drivers\etc\hosts”`.
```

Comment:
I don't think you need to say recent here. It was in that location with I believe every NT version of Windows (so anything past ME *shudder* for clients and Win 2000+ for server).